api/src/main/java/io/opentelemetry/trace/samplers/DefaultDecision.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.trace.samplers;
+
+import io.opentelemetry.trace.AttributeValue;
+import io.opentelemetry.trace.Sampler.Decision;
+import java.util.Collections;
+import java.util.Map;
+
+/** Sampling decision without attributes. */
+final class DefaultDecision implements Decision {
```

Comment:
Maybe call it `SimpleDecision`? `Default` does not feel right here as this instance can actually have a pair of values/cases (depending on the `isSampled` parameter).

Reply:
I didn't like `Default` bc we have `Default|tracer|tagger|meter`. Simple seems better.

I wanted to call it also `NoAttributesDecision`, `FlatDecision` something in those lines..

Reply:
As long as it is package protected and not exposed publicly I would not worry about the name that much. NoAtrributes is good as well as Simple.